src/utils/framework-server.ts

```diff
@@ -30,9 +31,11 @@ export const startFrameworkServer = async function ({
     if (settings.command) {
       runCommand(settings.command, { env: settings.env, cwd })
     }
-    await startStaticServer({ settings })
+    const { address } = await startStaticServer({ settings })
```

Comment:
Rather than getting `address` and then trying to figure out the IP family from the hostname, can you not use the `family` property directly?

https://nodejs.org/api/net.html#serveraddress

Reply:
I'm gonna find out!!! yes!

Reply:
✅